genie-core/src/main/java/com/netflix/genie/core/jpa/entities/JobRequestEntity.java

```diff
@@ -84,6 +84,11 @@
 
     @Lob
     @Basic
```

Comment:
If you're marking the database field as `TEXT NOT NULL` i think you need to have `@Basic(optional=false)` here to make it clear it's not nullable.  Same with `@Column(nullable=false)`